CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Unify CLI debug logging under a global `--verbose` flag ([#1361](https://github.com/operator-framework/operator-sdk/pull/1361))
 - [Go module](https://github.com/golang/go/wiki/Modules) support by default for new Go operators and during Ansible and Helm operator migration. The dependency manager used for a new operator can be explicitly specified for new operators through the `--dep-manager` flag, available in [`operator-sdk new`](https://github.com/operator-framework/operator-sdk/blob/master/doc/sdk-cli-reference.md#new) and [`operator-sdk migrate`](https://github.com/operator-framework/operator-sdk/blob/master/doc/sdk-cli-reference.md#migrate). `dep` is still available through `--dep-manager=dep`. ([#1001](https://github.com/operator-framework/operator-sdk/pull/1001))
 - New optional flag `--custom-api-import` for [`operator-sdk add controller`](https://github.com/operator-framework/operator-sdk/blob/master/doc/sdk-cli-reference.md#controller) to specify that the new controller reconciles a built-in or external Kubernetes API, and what import path and identifier it should have. ([#1344](https://github.com/operator-framework/operator-sdk/pull/1344))
+- Operator Scorecard plugin support ([#1379](https://github.com/operator-framework/operator-sdk/pull/1379)).
```

Comment:
Might want to elaborate a bit and link the scorecard plugin [design doc](https://github.com/operator-framework/operator-sdk/blob/master/doc/proposals/scorecard-plugin-system.md).

Reply:
I was originally going to link to the scorecard doc, but that's very out of date. The proposal design doc has some significant differences to some of the structures, so I feel like linking that would be a bad idea. Since the updated scorecard docs PR wasn't going to make it through today, I just decided to briefly include this in the changelog without linking to docs yet.

Docs PR: https://github.com/operator-framework/operator-sdk/pull/1433

Reply:
While not ideal, how about including a link to the docs update PR #1433 while we wait to get it merged after the release.
And once that docs PR is merged we can update the CHANGELOG to point to just the latest docs.

Reply:
While I don't like changing the changelog after a release, I think that's reasonable for this situation